backend/src/utils/image-utils.ts

```diff
@@ -0,0 +1,97 @@
+import { bucket } from '../firebase';
+import { getNetIDFromEmail } from './memberUtil';
+import { NotFoundError } from './errors';
+
+/**
+ * Sets image for member
+ * @param name - the name of the image
+ * @returns a Promise to the signed URL to the image file
+ */
+export const setImage = async (name: string): Promise<string> => {
+  const file = bucket.file(`${name}.jpg`);
+  const signedURL = await file.getSignedUrl({
+    action: 'write',
+    version: 'v4',
+    expires: Date.now() + 15 * 60000 // 15 min
+  });
+  return signedURL[0];
+};
+
+/**
+ * Gets image for member
+ * @param name - the name of the image
+ * @throws NotFoundError if the requested image does not exist
+ * @returns a Promise to the signed URL to the image file
+ */
+export const getImage = async (name: string): Promise<string> => {
+  const file = bucket.file(`${name}.jpg`);
+  const fileExists = await file.exists().then((result) => result[0]);
+  if (!fileExists) {
+    throw new NotFoundError(`The requested image (${name}) does not exist`);
+  }
+  const signedUrl = await file.getSignedUrl({
+    action: 'read',
+    expires: Date.now() + 15 * 60000
+  });
+  return signedUrl[0];
+};
```

Comment:
ok so let's rename these (ik i'm the one that gave them shitty names in the first place)
- `getWriteSignedURL`
- `getReadSignedURL` 

or something a long those lines